app/assets/stylesheets/public/_variables.scss

```diff
@@ -3,3 +3,5 @@ $text-on-dark-subdued-opacity: .8;
 $text-on-dark-very-subdued-opacity: .7;
 $color-brand-green: rgb(2, 129, 76);
 $transition-speed: 300ms;
+$screen-sm: 768px;
+$max-w-prose: 65ch;
```

Comment:
`65ch` width makes each line of text roughly 9–12 words, which is the optimal reading in tradition typography design.

This is inspired by TailwindCSS's `$prose` length. Setting character length is way simpler and interoperable than using JavaScript to set word width.